index.html

```diff
@@ -0,0 +1,100 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1.0"
+    />
+    <title>Marcelo Alarcon Portafolio</title>
+    <link
+      rel="icon"
+      type="image/x-icon"
+      href="images/codeoutlinedprogrammingsigns_81143.ico"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Inter:wght@300;400;500;600;700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <header>
+      <img src="images/Welcome.png" alt="logo" />
```

Comment:
-[Optional] Dear @chelobotix, I prefer putting your logo inside your navigation `nav`, This is because your logo is also a link that is used to navigate to the homepage. Also, please consider using text instead of an image for your logo. Using images over and over will make your site heavy to load.